frontend/src/pages/Securities.js

```diff
@@ -22,14 +24,15 @@ const noMatchesTextStyle = {
 const MAX_PAGE_WINDOW_SIZE = 7;
 
 const Securities = () => {
+    const dispatch = useDispatch();
```

Comment:
I think we should dispatch this action when we open the modal. This saves an additional network call when just opening the securities page.